site/docs/ch/docker-quickstart.md

```diff
@@ -126,7 +126,7 @@ CREATE TABLE cdc_source(
 INSERT INTO arctic.test_db.test_table
 SELECT id,
        name,
-       TO_TIMESTAMP(op_time) op_time
+       CAST(TO_TIMESTAMP(op_time) AS TIMESTAMP(6) WITH LOCAL TIME ZONE) op_time
```

Comment:
Is this field: op_time a timestamp with the time zone in the test_table?
```
create table test_db.test_table(
  id int,
  name string,
  op_time timestamp,
  primary key(id)
) using arctic partitioned by(days(op_time));
```

Reply:
Yes, After change in this PR, spark will create timestamp with zone column in test table.